app/controllers/scim_rails/application_controller.rb

```diff
@@ -238,34 +250,18 @@ def update_status(object)
         deprovision_method = ScimRails.config.group_deprovision_method
       end
 
-      object.public_send(reprovision_method) if active?
-      object.public_send(deprovision_method) unless active?
-    end
-
-    def active?
-      active = params[:active]
-
-      case active
-      when true, "true", 1
-        true
-      when false, "false", 0
-        false
-      else
-        raise ActiveRecord::RecordInvalid
+      if patch_status(params[:active])
+        object.public_send(reprovision_method) 
+      else 
+        object.public_send(deprovision_method)
       end
     end
 
     def patch_status(active_param)
-      case active_param
-      when true, "true", 1
-        true
-      when false, "false", 0
-        false
-      when nil
-        nil
-      else
-        raise ScimRails::ExceptionHandler::InvalidActiveParam
-      end
+      # Must use .to_s.downcase to handle strings "True" and "False"
+      # Which is sent by Azure AD
+      # Other possible inputs (true, "true", false, "false", 0, 1, nil)
+      ActiveModel::Type::Boolean.new.cast(active_param.to_s.downcase)
```

Comment:
Hmmm 💭  two thoughts on this.
1. I can't find tests around this... where can I find them?
2. We've lost `raise ScimRails::ExceptionHandler::InvalidActiveParam`... so 

```
[5] pry(main)> ActiveModel::Type::Boolean.new.cast({}.to_s.downcase)
=> true  
```

```
[7] pry(main)> ActiveModel::Type::Boolean.new.cast([].to_s.downcase)
=> true
```

these kinda values are also accepted now.

Reply:
I see your your point but as i put in the comments on the line above so far we know the possible inputs from various identity providers and what we know are possible inputs are `(true, "true", "True", false, "false", "False", 0, 1, nil)`. So based on this I still think its an adequate solution